release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1598,6 +1598,156 @@ If these values are not defined in a failure domain configuration, for instance
+//------ page 1
+* Previously, when you specified the `HostedZoneRole` permission in the `install-config.yaml` file while creating a shared Virtual Private Cloud (VPC), you also had to specify the `sts:AssumeRole` permission. Otherwise, it caused an error. With this release, if you specify the `HostedZoneRole` permission the installer validates that the `sts:AssumeRole` permission is present.
+(link:https://issues.redhat.com/browse/OCPBUGS-46046[*OCPBUGS-46046*])
+
+* Previously, when the `publicIpv4Pool` configuration parameter was used during installation the permissions `ec2:AllocateAddress` and `ec2:AssociateAddress` were not validated. As a consequence, permission failures might occur during installation. With this release, the required permissions are validated before the cluster is installed and the issue is resolved.(link:https://issues.redhat.com/browse/OCPBUGS-45711[*OCPBUGS-45711*])
+
+* Previously, during a disconnected installation, when the `imageContentSources` parameter was configured for more than one mirror for a source, the command to create the agent ISO image could fail, depending on the sequence of the mirror configuration. With this release, multiple mirrors are handled correctly when the agent ISO is created and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-45630[*OCPBUGS-45630*])
+
+* Previously, during CAPI installer-provisioned installation, the user provided a `machineNetwork` parameter. With this release, the installer uses a random `machineNetwork` parameter.
+(link:https://issues.redhat.com/browse/OCPBUGS-45485[*OCPBUGS-45485*])
+
+* Previously, during an installation on {aws-first}, the installer used the wrong load balancer when searching for the `hostedZone` ID, which caused an error. With the release, the correct load balancer is used and the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-45301[*OCPBUGS-45301*])
+
+* Previously, endpoint overrides in {ibm-power-server-title} were not conditional. As a consequence, endpoint overrides were created incorrectly and caused failures in Virtual Private Environments (VPEs). With this release, endpoint overrides are conditional only for disconnected installations.
+(link:https://issues.redhat.com/browse/OCPBUGS-44922[*OCPBUGS-44922*])
+
+* Previously, during a shared Virtual Private Cloud (VPC) installation, the installer added the records to a private DNS zone created by the installer instead of adding the records to the cluster’s private DNS zone. As a consequence, the installation failed. With this release, the installer searches for an existing private DNS zone and, if found, pairs that zone with the network that is supplied by the `install-config.yaml` file and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44641[*OCPBUGS-44641*])
```

Comment:
```suggestion
* Previously, during a shared Virtual Private Cloud (VPC) installation, the installation program added the records to a private DNS zone created by the installation program instead of adding the records to the cluster’s private DNS zone. As a consequence, the installation failed. With this release, the installation program searches for an existing private DNS zone and, if found, pairs that zone with the network that is supplied by the `install-config.yaml` file and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44641[*OCPBUGS-44641*])
```
Having 'installation program' so close together might sound a bit clunky...just for your consideration. :-)